test/scripts/run_test.py

```diff
@@ -158,13 +209,17 @@ def recover_deps():
     subprocess.call(cmd)
 
 
+def recover_unit_test_environment():
+    for module, unit_test in UNIT_TESTS.items():
+        shutil.rmtree(os.path.join(unit_test['path'], 'src/androidTest/jniLibs'))
+
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Run android instrumentation tests.')
     parser.add_argument('--build-deps', dest='build', action='store_true', default=False,
                         help='Indicates if to build sdk libraries.')
-    parser.add_argument('--caselist', dest='caselist',
-                        default=os.path.join(TEST_PATH, 'case_list.json'),
-                        help='Location of the case list json file.')
+    parser.add_argument('--instrumentation', dest='instrumentation', default=False,
+                        help='Run instrumentation test, parameter:location of the case list json file.')
```

Comment:
`default=False` doesn't make sense.

Reply:
done

Reply:
What if instrumentation test is not supposed to run?

Reply:
default = False , if instrumentation test is not supposed to run , test script will not run  instrumentation

Reply:
`default=False`? apparently the code is not as you claimed.

Reply:
the first commit i do it, but the last commit because default=os.path.join(TEST_PATH, 'case_list.json') so i remove it

Reply:
so i asked 

> What if instrumentation test is not supposed to run?

Reply:
use default=os.path.join(TEST_PATH, 'case_list.json') :
set --instrumentation and path is not correct 
not set --instrumentation and os.path.join(TEST_PATH, 'case_list.json') not exist

use default=Fasle:
set --instrumentation and path is not correct
not set --instrumentation

Reply:
Relying on the nonexistence of `case_list.json` not to run instrumentation test is not a good design, from my perspective.

Reply:
run instrumentation test need .json file ,so how can i design if not run instrumentation test 
can i design  : --instrumentation  --caselist-path , if --instrumentation not set ,it will not run instrumentation

Reply:
Multiple ways.
but implementing this script is not my job, otherwise i would file a PR for this.